### PR TITLE
chore: relative links for contribution guidelines

### PR DESCRIPTION
**Pull request by lmilbaum**
## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by rarkins** [comment]
Thanks for the PR. How confident are you that these relative links work in every case?

**Comment by lmilbaum** [comment]
> Thanks for the PR. How confident are you that these relative links work in every case?

I am confident with the md files. Not confident with the others. Didn't figure out yet how to validate them. Any idea?

**Comment by rarkins** [comment]
We can maybe manually edit some existing ones and test if it still works correctly afterwards

**Comment by lmilbaum** [comment]
Shell I close the PR?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!-- If this is your first pull request: sign the CLA with this GitHub app: https://cla-assistant.io/renovatebot/renovate -->
 <!-- Make sure the `Allow edits and access to secrets by maintainers` checkbox is checked on this pull request. -->
-<!-- Please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md before you create your pull request.-->
+<!-- Please read [contributing guidelines](.github/contributing.md) before you create your pull request.-->
```
Comment (rarkins) [inline .github/pull_request_template.md:3]:
```suggestion
<!-- Please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md before you create your pull request.-->
```

File: .github/label-actions.yml [hunk 2]
```diff
@@ -158,7 +158,7 @@
 
     You are using simple `done` comments which cause a lot of noise.
     Instead, please use GitHub's web interface to request another review.
-    Please read our [contribution guide](https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting) to reduce noise.
+    Please read our [contributing guidelines](.github/contributing.md#resolve-review-comments-instead-of-commenting) to reduce noise.
```
Comment (rarkins) [inline .github/label-actions.yml:161]:
```suggestion
    Please read our [contributing guidelines](https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting) to reduce noise.
```

**Review by rarkins — CHANGES_REQUESTED** [review]

**Review by rarkins — DISMISSED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**